engine/time/src/main/java/io/deephaven/time/DateTimeUtils.java

```diff
@@ -3170,6 +3170,26 @@ public static Instant lowerBin(@Nullable final Instant instant, long intervalNan
         return epochNanosToInstant(Numeric.lowerBin(epochNanos(instant), intervalNanos));
     }
 
+    /**
+     * Returns an {@link Instant} value, which is at the starting (lower) end of a time range defined by the interval
+     * nanoseconds. For example, a 5*MINUTE intervalNanos value would return the instant value for the start of the
```

Comment:
I don't think `5*MINUTE intervalNanos` is correct in this context; maybe just `5 minute {@code interval}`?

Reply:
Or use {@code 5 * MINUTE}